### DISCUSSION TITLE
Search attribute in channel:entries doesn't work with tags, only static hardcoded values

### DISCUSSION THREAD

**Discussion by hockeymikey** [ExpressionEngine/ExpressionEngine#2976] [Feature Requests]
**Description of the problem**
I'm using the following tag in a template:
{exp:channel:entries show_future_entries="yes" dynamic="no" channel="news" search:end_page="{value}"}
It returns no values when the tag is used in the search attitbute. But, when I replace it with the actual number (say 63) it works. I've tried many other ways like using embedded values, or a value given from a parent tag surrounding the channel:entries tag with no luck.

**How To Reproduce**
1. Edit template to include channel:entries like show above.
2. Add search attribute for some text field that is searchable and use some value, not static

People don't seem to have this issue, but their posts were very old. Addition to this, this channel:entries tag is nested in another one, kinda. It's an embed which is called from the parent. I don't know why that would cause issues.
-->

**Error Messages**

**Screenshots / Videos / Template Code**
I could detail more if not clear but:
parent exp:channel:entries -> embed/blah (which contains the troubled tag in question with the search)

**Environment Details:**
 - Version: 5.2.0 and 5.3.0
 - PHP Version 7.2 and 7.4
 - MySQL 5.7.3 
 - OS: CentOS 7
 - Web Server: Apache (cpanel)

**Comment by hockeymikey** [comment]
This shouldn't be a discussion, this is a bug.

**Reply by hockeymikey** [reply]
@intoeetive

**Reply by intoeetive** [reply]
While the behaviour here does not match your expectations, it is rather a limitation, not bug as such.

Changing this behaviour would require significant code updates and might involve changes in multiple places, so we're classifying it as feature request.

That does not make it less important. We realise that people want it to be able to work the way you describe
